docs/pages/reference/join-methods.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Join Methods and Token Reference
+description: Describes the different ways to configure a node to join the Teleport cluster.
```

Comment:
Should we say "instance" or "service" instead of node?

Reply:
Agent does not fit here because non-agent Teleport instances also join (e.g. Proxies), and some instances joining are not even running the teleport binary (tbot, the operator).

I think instance is the least problematic term.